src/ClusterBootstrap/services/repairmanageragent/repairmanageragent.yaml

```diff
@@ -0,0 +1,90 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: repairmanageragent
+  namespace: default
+spec:
+  selector:
+    matchLabels:
+      repairmanageragent-node: pod
+  template:
+    metadata:
+      name: repairmanageragent
+      labels:
+        repairmanageragent-node: pod
+        app: repairmanageragent
+    spec:
+      {% if cnf["dnsPolicy"] %}
+      dnsPolicy: {{cnf["dnsPolicy"]}}
+      {% endif %}
+      hostPID: true
+      nodeSelector:
+        repairmanageragent: active
+      containers:
+      - name: repairmanageragent
+        image: {{cnf["worker-dockerregistry"]}}/{{cnf["dockerprefix"]}}/{{cnf["repairmanageragent"]}}:{{cnf["dockertag"]}}
+        securityContext:
+          capabilities:
+            add: ["SYS_BOOT"]
```

Comment:
Added SYS_BOOT linux capability to allow for reboot syscall from docker container